### PR TITLE
modularized service_repository and service_tool resources

### PR DESCRIPTION
**Pull request by davidbloss**

### CODE REVIEW

File: modules/service/repository/variables.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+variable "base_directory" {
+  type        = string
+  description = "The directory in the repository containing opslevel.yml."
+  default     = null
+}
+
+variable "name" {
+  type        = string
+  description = "The name displayed in the UI for the service repository."
+  default     = null
```
Comment (rocktavious) [inline modules/service/repository/variables.tf:10]: This is required i think

File: modules/service/repository/variables.tf [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+variable "repository_alias" {
+  type        = string
+  description = "The alias of the repository that this will be added to."
+  default     = null
+}
+
+variable "service" {
+  type        = string
+  description = "The id of the service that this will be added to."
+  default     = null
```
Comment (rocktavious) [inline modules/service/repository/variables.tf:28]: this is also required

**Review by rocktavious — APPROVED** [review]
LGTM - just one minor change

File: modules/service/repository/variables.tf (hunk 1 above)
Comment (davidbloss) [reply modules/service/repository/variables.tf:10]: This is optional in the actual [schema](https://github.com/OpsLevel/terraform-provider-opslevel/blob/main/opslevel/resource_opslevel_service_repository.go#L93-L96), but should it be required?

Comment (davidbloss) [reply modules/service/repository/variables.tf:10]: I just checked the API schema and this is for the displayName which is optional

**Pull request merged**